src/pages/index.js

```diff
@@ -102,13 +288,40 @@ function handleAddCardFormSubmit(inputValues) {
 //   addCardFormElement.reset();
 // }
 
+addNewCardButton.addEventListener("click", () => {
+  addCardPopup.open();
+});
+
+// EDIT PROFILE MODAL AND METHODS
+const editProfilePopup = new PopupWithForm(
+  "#profile-edit-modal",
+  handleProfileEditSubmit,
+  editFormValidator
+);
+editProfilePopup.setEventListeners();
+
+// since setUserInfo is expecting an object with name property and description property, use name & description key/value pairs with formData inside {}
+function handleProfileEditSubmit(formData) {
+  // console.log(formData);
+  function makeRequest() {
+    // using the response from the API (res) to set the user info
+    return api.updateProfileInfo(formData).then((res) => {
+      //res is an object with name, about and avatar properties, so you have to use it while setting the user info:
+      // console.log(res);
+      userInfo.setUserInfo({
+        name: res.name,
+        description: res.about,
+        avatar: res.avatar,
+      });
```

Comment:
* When I edit the profile data, the `job` disappears from the page https://snipboard.io/lizvkW.jpg


`res` is a ready object from now on. You can set it easily


```js
    return api.updateProfileInfo(formData).then((res) => {
      userInfo.setUserInfo(res);
    });

```